src/creatures/players/player.cpp

```diff
@@ -628,6 +629,19 @@ phmap::flat_hash_map<uint8_t, std::shared_ptr<Item>> Player::getAllSlotItems() c
 	return itemMap;
 }
 
+phmap::flat_hash_map<Blessings_t, std::string> Player::getBlessingNames() const {
```

Comment:
You need to set the map as static, otherwise every time a player is created, a new map will be created, which could be potentially bad

```cpp
    static phmap::flat_hash_map<Blessings_t, std::string> blessingNames = {
        { TWIST_OF_FATE, "Twist of Fate" },
        { WISDOM_OF_SOLITUDE, "The Wisdom of Solitude" },
        { SPARK_OF_THE_PHOENIX, "The Spark of the Phoenix" },
        { FIRE_OF_THE_SUNS, "The Fire of the Suns" },
        { SPIRITUAL_SHIELDING, "The Spiritual Shielding" },
        { EMBRACE_OF_TIBIA, "The Embrace of Tibia" },
        { BLOOD_OF_THE_MOUNTAIN, "Blood of the Mountain" },
        { HEARTH_OF_THE_MOUNTAIN, "Heart of the Mountain" },
    };
    
    return blessingNames;
    ```